modules/rosa-getting-started-configure-an-idp.adoc

```diff
@@ -8,6 +8,11 @@
 
 You can configure different identity provider types for your {product-title} (ROSA) cluster. Supported types include GitHub, GitHub Enterprise, GitLab, Google, LDAP, OpenID Connect and HTPassword identity providers.
```

Comment:
Please replace "HTPassword" with "HTPasswd".

Reply:
Good catch, thank you! Resolved.